src/main/java/io/dropwizard/sharding/dao/RelationalDao.java

```diff
@@ -275,7 +275,7 @@ public boolean exists(String parentKey, Object key) throws Exception {
     }
 
     public List<T> scatterGather(DetachedCriteria criteria) {
-        return daos.stream().map(dao -> {
+        return daos.stream().parallel().map(dao -> {
```

Comment:
Don't do parallel. It uses common thread pool. ScatterGather is not the primary usecase anyways.